build/bin/build_mfsroot

```diff
@@ -183,14 +183,11 @@ ${INSTALL_DEF_LIB} ${X_DESTDIR}/lib/libdevstat.so.7 ${X_STAGING_FSROOT}/lib/
 ${INSTALL_DEF_BIN} ${X_DESTDIR}/usr/sbin/devinfo ${X_STAGING_FSROOT}/usr/sbin/
 ${INSTALL_DEF_LIB} ${X_DESTDIR}/usr/lib/libdevinfo.so.6 ${X_STAGING_FSROOT}/usr/lib/
 
-# libraries are for grep, tar, cpio, df
-${INSTALL_DEF_BIN} ${X_DESTDIR}/usr/bin/tar ${X_STAGING_FSROOT}/usr/bin/
-${INSTALL_DEF_BIN} ${X_DESTDIR}/usr/bin/cpio ${X_STAGING_FSROOT}/usr/bin/
+# libraries are for grep, df
 ${INSTALL_DEF_BIN} ${X_DESTDIR}/usr/bin/grep ${X_STAGING_FSROOT}/usr/bin/
 ${INSTALL_DEF_LIB} ${X_DESTDIR}/usr/lib/libgnuregex.so.5 ${X_STAGING_FSROOT}/usr/lib/
```

Comment:
Do you use GNU extensions? I'd suspect not, in which case there's another knob for GNU_GREP_COMPAT (iirc) and we could slice off this one, too, though it's pretty minimal.